src/seedu/addressbook/AddressBook.java

```diff
@@ -450,8 +450,8 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
      * @return feedback display message for the operation result
      */
     private static String executeFindPersons(String commandArgs) {
-        final Set<String> keywords = extractKeywordsFromFindPersonArgs(commandArgs);
-        final ArrayList<String[]> personsFound = getPersonsWithNameContainingAnyKeyword(keywords);
+        final Set<String> keywords = extractKeywordsFromFindPersonArgsInLowerCase(commandArgs);
+        final ArrayList<String[]> personsFound = getPersonsWithNameContainingAnyCaseInsensitiveKeyword(keywords);
```

Comment:
I was thinking that case keyword would be a little bit still rather vague. Thanks for the input, I do feel the method name is a little bit too long haha